### PR TITLE
Article about the album Bella Donna

### PR DESCRIPTION
**Pull request by sonjazw**
The first album of Stevie Nicks

Please consider adding this article to the Billboard

### CODE REVIEW

**Comment by claudiodsf** [comment]
Tank you sonja. Looks nice

File: index.html [hunk 1]
```diff
@@ -24,6 +24,7 @@
 
   <div class="columns">
     <div class="column">
+      <div id="example"><script>include("articles/belladonna.md", "example")</script></div>
```
Comment (claudiodsf) [inline index.html]: Please replace `example`

Comment (sonjazw) [reply index.html]: I have solved the problem.

**Comment by claudiodsf** [comment]
Squashed into one commit and merged!

**Pull request merged**